packages/suite-desktop-api/src/api.ts

```diff
@@ -56,8 +60,9 @@ export interface RendererChannels {
 // Sent by DesktopApi.[method] via ipcRenderer.invoke (./main)
 // Handled by ipcMain.handle (see packages/suite-desktop/src-electron/modules/*)
 export interface InvokeChannels {
-    'handshake/client': () => void;
+    'handshake/client': () => InitialHandShakeClient;
     'handshake/load-modules': (payload: HandshakeClient) => InvokeResult<HandshakeElectron>;
+    'handshake/load-tor': (payload: HandshakeClient) => HandsakeTorLoad;
```

Comment:
just wondering why you did not use `InvokeResult` generic here?

Reply:
Maybe it could work with generic `InvokeResult`.

Reply:
Yes it did not make much sense before, because it really was very similar to generic `InvokeResult` but now it was changed to return `{shouldRunTor: boolean}` in order to be able not to display the TorSplashScreen when Tor is disable or when reloading.